utils/handler-utils.js

```diff
@@ -76,3 +76,15 @@ module.exports.log = function log() {
 };
 
 module.exports.getSecret = secrets => secret => secrets.find(s => s.key === secret).value;
+
+module.exports.fixBuffers = (d) => {
```

Comment:
What's this for?

Reply:
When a file gets passed to lambda, it gets stringified into that weird buffer string format, so we have to convert it back to being a buffer